sympy/physics/units/util.py

```diff
@@ -49,6 +49,7 @@ def _get_conversion_matrix_for_expr(expr, target_units, unit_system):
 
 
 def convert_to(expr, target_units, unit_system="SI"):
+    expr=expr.simplify()
     """
     Convert ``expr`` to the same expression with all of its units and quantities
```

Comment:
We generally don't want to use `simplify` in library code. Also there shouldn't be code before a docstring.

Reply:
Thankyou for acknowledging , but checking if the `dir(expr)` has simplify in it then the expr is having simplified version of itself . so `simplify `function was helping to get out of the unit that it wanted and in future PR it adding code before docstring will not be happen .

Reply:
@oscarbenjamin I have tried to check whether simplify is present in expr if it is present then it simplifies the expression when I run through sample of test cases as well as the test case where unit was not shown up , it started working well
![Screenshot (187)](https://user-images.githubusercontent.com/70062727/170706897-8def9431-c0f9-4560-a7ac-96a0f9074173.png)
so is there any alternate way to add simplify?, as when adding simplify function to util makes the test case to pass and unit was also showing up 

This patch was added to get accepted behavior in util.py 
```
if 'simplify' in dir(expr):
        expr=expr.simplify()
```

Reply:
We don't want to use `simplify` here.